core/src/test/java/io/questdb/cairo/CreateTableTest.java

```diff
@@ -161,6 +161,69 @@ public void testCreateTableAsSelectInheritsColumnIndex() throws Exception {
         assertColumnsIndexed("new", "s");
     }
 
```

Comment:
Tests are missing for the following: WAL enabled flag, symbol cache flag, column types, maxUncommittedRows, commitLag. Would be also nice to test `CREATE TABLE x IF NOT EXISTS (LIKE y)` variation of the DDL to make sure that it works as expected.